.github/workflows/lint-and-test.yml

```
@@ -8,15 +8,17 @@ jobs:
     name: Lint and test
     runs-on: ubuntu-latest
     steps:
-      - name: Checkout
+      - name: Check out project repo under $GITHUB_WORKSPACE
         uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Setup node
+      - name: Set up Node.js runtime
         uses: actions/setup-node@60edb5dd545a775178f52524783378180af0d1f8 # v4.0.2
-      - name: Setup Ruby, JRuby, and TruffleRuby
+        with:
+          node-version-file: 'package.json'
+      - name: Set up Ruby runtime
         uses: ruby/setup-ruby@5f19ec79cedfadb78ab837f95b87734d0003c899 # v1.173.0
-      - name: Install
+      - name: Install & configure language runtime PMs & the deps they manage
```

```suggestion
      - name: Install
```

we will break all the below down in a follow-up